test/api/v3/integration/groups/POST-groups.test.js

```diff
@@ -189,6 +189,17 @@ describe('POST /group', () => {
       expect(updatedUser.party._id).to.eql(party._id);
     });
 
+    it('does not award Party Up achievement to solo partier', async () => {
+      await user.post('/groups', {
+        name: partyName,
+        type: partyType,
+      });
+
+      let updatedUser = await user.get('/user');
```

Comment:
This is the same pattern used in tests throughout the file, though. Do we want to update the rest of the tests to use that approach?